src/nft/hooks/useSubscribeScrollState.ts

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react'
+
+export function useSubscribeScrollState() {
+  const [userCanScroll, setUserCanScroll] = useState(false)
+  const [scrollProgress, setScrollProgress] = useState(0)
+  const scrollRef = (node: HTMLDivElement) => {
+    if (node !== null) {
+      const canScroll = node.scrollHeight > node.clientHeight
+      canScroll !== userCanScroll && setUserCanScroll(canScroll)
+    }
+  }
+
+  const scrollHandler = (event: React.UIEvent<HTMLDivElement>) => {
```

Comment:
Did you copy this from `nft/components/Bag`? If yes, why not to reuse that hook there too?

Reply:
Sounds good to me! If you could either have a JIRA ticket for it or just write it down somewhere on a list of follow-up tickets so it doesn't get lost - it would be appreciated. Feel free to CC me on that PR when it's up.

Reply:
The ticket is tagged at the top of the hook `TODO(NFT-1190)`